spec/atc/aws/fixity_check_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Atc::Aws::FixityCheck do
+  subject(:aws_fixity_check) { described_class.new(aws_stored_object, 3141) }
+
+  let(:aws_storage_provider) { FactoryBot.create(:storage_provider, container_name: 'AWS bucket', storage_type: 0) }
+  let(:checksum_algorithm) { FactoryBot.create(:checksum_algorithm, name: 'SHA31415', empty_binary_value: 0) }
+  let(:source_object) do
+    FactoryBot.create(:source_object,
+                      fixity_checksum_algorithm: checksum_algorithm,
+                      fixity_checksum_value: 'ABCDEF12345',
+                      object_size: 4321)
+  end
+  let(:aws_stored_object) do
+    FactoryBot.create(:stored_object,
+                      source_object: source_object,
+                      storage_provider: aws_storage_provider)
+  end
+  let(:aws_hash_response) do
+    { 'type' => 'fixity_check_complete',
+      'data' => { 'checksum_hexdigest' => 'ABCDEF12345', 'object_size' => 1234 } }
+  end
+  let(:aws_error_hash_response) do
+    { 'type' => 'fixity_check_error',
+      'data' => { 'error_message' => 'Ooops!',
+                  'job_identifier' => 1234,
+                  'bucket_name' => 'cul_bucket',
+                  'object_path' => '/I/Am/An/Object',
```

Comment:
```suggestion
                  'object_path' => 'I/Am/An/Object',
```
(just because our expected object paths won't start with a slash)